modules/openideal_idea/openideal_idea.group.permissions.yml

```diff
@@ -0,0 +1,2 @@
+vote on five stars:
```

Comment:
The fewer words, the better it is. First of all, let's use `fivestar` instead of `five starts`. As I see, the widget ID is "fivestar". So why won't we use this name?

I find the permission name confusing. Wouldn't it be better to use something like:
1) use fivestar widget;
2) vote on expert review;